src/openforms/contrib/zgw/api/serializers.py

```diff
@@ -3,11 +3,11 @@
 from rest_framework import serializers
 
 
-class CatalogusSerializer(serializers.Serializer):
-    domein = serializers.CharField(label=_("domein"))
+class CatalogueSerializer(serializers.Serializer):
+    domain = serializers.CharField(label=_("domain"))
     rsin = serializers.CharField(label=_("rsin"))
-    label = serializers.CharField(
-        label=_("label"), help_text=_("The display label of the catalogus")
+    label = serializers.CharField(  # type: ignore
```

Comment:
Why was this `type: ignore` not needed before?

Reply:
it was, it was just not added yet :) it's because serializer inherits from `Field`, which has a `label` attribute of type (lazy) string - which doesn't match with the type used here being `serializers.CharField`.

The type checker doesn't understand the serializer meta class that processes this, and errors because of that.